### ISSUE TITLE
[Bug]: Pagination Overflow select element has incorrect background color in firefox

### ISSUE THREAD

**Issue by Akshat55** [carbon-design-system/carbon#11280]
### Package

@carbon/react, @carbon/styles

### Browser

Firefox

### Package version

1.1.0

### React version

17.0.2

### Description

In firefox, Select dropdown list background for pagination nav appears gray & the text color is black making it difficult to tell the options apart and not readable.

***Issue does not occur in chromium based browsers, I have no checked for this issue on safari**

![image](https://user-images.githubusercontent.com/38994122/165022914-e7787107-ee0e-485f-be4a-2023b7a0922e.png)

I'm assuming white is the default theme so I don't need to specify the theme 🤔 

### CodeSandbox example

https://codesandbox.io/s/strange-montalcini-fzxojq?file=/src/App.js

### Steps to reproduce

1. Set itemShown (5) to lower than totalItems (10) which brings up the overflow 
2. Open overflow select drop down - You'll see select options have a gray background
4. Hover over any option and you'll see the text turn white

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by abbeyhrt** [comment]
Hi @Akshat55! Thanks for making this issue!  Are you still encountering this problem? 

I wasn't able to reproduce it, the code sandbox you linked unfortunately wasn't working for me but I followed your example in a stackblitz in firefox and the select appeared correct for me https://stackblitz.com/edit/github-p6nzho?file=src/App.jsx but I wanted to confirm with you before closing the issue

**Comment by Akshat55** [comment]
Hey @abbeyhrt, I cannot run stackblitz on firefox nor the container in a separate tab. I checked locally & it works in white theme now.

However, in `g100` & `g90` it does not & this time it doesn't work in chromium based browsers as well. [Stackblitz link](https://stackblitz.com/edit/github-nxd2f8-3x7vww?file=src%2FApp.jsx)

Firefox local:
![image](https://user-images.githubusercontent.com/38994122/193165342-0e480cf5-be42-473d-8fee-8418215771e6.png)

Chrome:
![image](https://user-images.githubusercontent.com/38994122/193165671-d5bcbde7-f052-461b-b311-9be289435a42.png)

**Comment by abbeyhrt** [comment]
Hmm do you have any browser or systems preferences on for color contrast? We are using a native html element, without any styles for those select options so it would be up to the browsers/systems to style them.  I think these _could_ be browser bugs, I'm seeing some weird behavior in firefox with just a `select` html element where colors flash and the text color changes on hover/select as well

**Comment by Akshat55** [comment]
That could be the case, I'm using Windows 11 with dark theme. My browser theme for both chrome & Firefox is set to dark.

**Comment by tay1orjones** [comment]
I think this bug on firefox might be the culprit here https://bugzilla.mozilla.org/show_bug.cgi?id=1617429

I'm going to close and mark this as a wontfix since we're blocked by the browser.
